drivers/SmartThings/zigbee-window-treatment/src/aqara/aqara_utils.lua

```diff
@@ -15,13 +15,23 @@ local SHADE_STATE_STOP = 0
 local SHADE_STATE_OPEN = 1
 local SHADE_STATE_CLOSE = 2
 
+local UPDATE_SHADE_LEVEL = "update_shade_level"
 local PREF_REVERSE_DEFAULT = "\x00\x02\x00\x00\x00\x00\x00"
 local PREF_REVERSE_OFF = "\x00\x02\x00\x00\x00\x00\x00"
 local PREF_REVERSE_ON = "\x00\x02\x00\x01\x00\x00\x00"
 
 local aqara_utils = {}
 
+local function enable_update_shade_level(device)
+  device:set_field(UPDATE_SHADE_LEVEL, true, { persist = true })
+end
+
+local function disable_update_shade_level(device)
+  device:set_field(UPDATE_SHADE_LEVEL, false, { persist = true })
+end
+
```

Comment:
This value doesn't need to be persistent. The useful timeline of this flag is from when the first command is issued until when the stop report is received. Having this value persist during a hub reboot would probably lead to unintended side-effects, especially if the "stop" report is missed.

Reply:
I see what you mean.